src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -607,25 +372,126 @@ public void highlightKeyStringInMessageInfoHtml() {
         }
         
         for (String stringToHighlight : keyStringsToHighlight) {
-            if (message.toLowerCase().contains(stringToHighlight.toLowerCase())) {
+            if (logMessage.toLowerCase().contains(stringToHighlight.toLowerCase())) {
                 
-                int startIndex = message.toLowerCase().indexOf(stringToHighlight.toLowerCase());
+                int startIndex = logMessage.toLowerCase().indexOf(stringToHighlight.toLowerCase());
                 int endIndex = startIndex + stringToHighlight.length();
-                String realStringToHighlight = message.substring(startIndex, endIndex);
-                message = message.replace(realStringToHighlight, "<mark>" + realStringToHighlight + "</mark>");
+                String realStringToHighlight = logMessage.substring(startIndex, endIndex);
+                logMessage = logMessage.replace(realStringToHighlight, "<mark>" + realStringToHighlight + "</mark>");
             }
         }
         
-        logInfoAsHtml = getLogInfoForTableRowAsHtml();
-        
     }
     
     public void setFirstRow() {
         isFirstRow = true;
     }
     
-    public boolean isTestingData() {
-        return email.endsWith(".tmt");
+    /**
+     * A builder class for {@link ActivityLogEntry}.
+     * <p>
+     * All optional fields are initialized to {@link Const.ActivityLog.UNKNOWN}.
+     * Field actionResponse will be initialized to have the same value as actionName.
+     * All null values (if possible) that are passed into the builder will be ignored.
+     * 
+     * @see {@link ActivityLogEntry}
```

Comment:
As mentioned in #6583, `@see` attaches link to the entry automatically so there is no need to `@link` it.